docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md

```diff
@@ -0,0 +1,171 @@
+og_description: "Pacman is the default package manager on Arched-based Linux distributions. This guide walks you through the core features of Pacman and common commands for using Pacman to install, upgrade, and remove packages."
+keywords: ['arch','manjaro','package manager']
+tags: ['arch', 'manjaro']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-07-15
+modified: 2021-07-15
+modified_by:
+  name: Linode
+title: "How to Manage Packages in Arch Using Pacman"
+h1_title: "Using Pacman to Manage Packages in Arch"
+enable_h1: true
+---
+
+Arch Linux uses binary packages in a `.tar.xz` format, and also provides a "ports" build system that facilitates building packages. Arch Linux runs on a *rolling release* schedule, which means packages are added to the main repository when they (and their dependencies) are ready for production. This means that there aren't release versions of Arch, as all systems, once upgraded, are equivalent. Therefore, administrators of Arch Linux must consider the output of `pacman` carefully before agreeing to upgrade or update any packages.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running Arch Linux** or an Arch-based distribution like Manjaro. Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
+
+1. **Login credentials to the system** for either the root user (not recommended) or a standard user account (belonging to the `sudo` group) and the ability to access the system through [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-linode-shell-lish/). Review the [Securing Your Server](/docs/guides/securing-your-server/) guide for assistance on creating and securing a standard user account.
+
+{{< note >}}
+Some commands in this guide require elevated privileges and are prefixed with the `sudo` command. If you are logged in as the root use (not recommended), you can omit the `sudo` prefix if desired. If you’re not familiar with the `sudo` command, see the [Linux Users and Groups](/docs/guides/linux-users-and-groups/#understanding-the-sudo-linux-group-and-user) guide.
+{{< /note >}}
+
+## Pacman
+
+The `pacman` tool is very powerful, but it is also very simple. There are three core commands for basic package management:
+
+-   `pacman --query [package]` or `pacman -Q`
+
+    Searches the package database for a package name and version number.
+
+-   `pacman --sync [package]` or `pacman -S`
+
+    Installs new packages, downloads new content for the database and/or upgrades the system, depending on the options and the named package or packages.
+
+-   `pacman --remove [package]` or `pacman -R`
+
+    Removes the named package or packages. Note that the terse flags are all uppercase and case-sensitive. These terse flags are often combined with additional flags for additional functionality. Here are some examples with brief descriptions:
```

Comment:
I feel like we should break the list here, have a standalone sentence about the terse flags, then start a new list with the other options.

Reply:
Let me know if my revisions were what you intended. Essentially put a paragraph break after that first sentence and modified the rest as follows:

> Each of the above single character flags (`-Q`, `-S`, and `-R`) are uppercase. They are often combined with additional flags (also case-sensitive) to alter the functionality of the command. Here are some examples with brief descriptions: